tests/unit/store/auth0.spec.js

```diff
@@ -0,0 +1,1081 @@
+          TypeError
+        )
+        expect(() =>
+          store.commit('isRefreshingAuthentication', undefined)
+        ).toThrow(TypeError)
+        expect(() =>
+          store.commit('isRefreshingAuthentication', 'test')
+        ).toThrow(TypeError)
+        expect(() => store.commit('isRefreshingAuthentication', 12345)).toThrow(
+          TypeError
+        )
+      })
+
+      it('isRefreshingAuthorization throws error on invalid state', () => {
+        expect(() => store.commit('isRefreshingAuthorization', null)).toThrow(
+          TypeError
+        )
+        expect(() =>
+          store.commit('isRefreshingAuthorization', undefined)
+        ).toThrow(TypeError)
+        expect(() => store.commit('isRefreshingAuthorization', 'test')).toThrow(
+          TypeError
+        )
+        expect(() => store.commit('isRefreshingAuthorization', 12345)).toThrow(
+          TypeError
+        )
+      })
+
+      it('isAuthorizingUser throws error on invalid state', () => {
+        expect(() => store.commit('isAuthorizingUser', null)).toThrow(TypeError)
+        expect(() => store.commit('isAuthorizingUser', undefined)).toThrow(
+          TypeError
+        )
+        expect(() => store.commit('isAuthorizingUser', 'test')).toThrow(
+          TypeError
+        )
+        expect(() => store.commit('isAuthorizingUser', 12345)).toThrow(
+          TypeError
+        )
+      })
+
+      it('isLoggingInUser throws error on invalid state', () => {
+        expect(() => store.commit('isLoggingInUser', null)).toThrow(TypeError)
+        expect(() => store.commit('isLoggingInUser', undefined)).toThrow(
+          TypeError
+        )
+        expect(() => store.commit('isLoggingInUser', 'test')).toThrow(TypeError)
+        expect(() => store.commit('isLoggingInUser', 12345)).toThrow(TypeError)
+      })
+
+      it('redirectRoute throws error on invalid state', () => {
+        expect(() => store.commit('redirectRoute', null)).toThrow(TypeError)
+        expect(() => store.commit('redirectRoute', false)).toThrow(TypeError)
+        expect(() => store.commit('redirectRoute', undefined)).toThrow(
+          TypeError
+        )
+        expect(() => store.commit('redirectRoute', 12345)).toThrow(TypeError)
+      })
+    })
+
+    describe('... successfully resets stores with unset methods', () => {
+      let store
+
+      beforeEach(() => {
+        store = new Vuex.Store({
+          state: auth0.state,
+          getters: auth0.getters,
+          actions: auth0.actions,
+          mutations: auth0.mutations
+        })
+      })
+
+      it('unsets user', () => {
+        store.commit('user', loggedInState().user)
+        expect(store.getters['user'].name).toBe(loggedInState().user.name)
+        expect(store.getters['user'].sub).toBe(loggedInState().user.sub)
+        expect(store.getters['user'].email).toBe(loggedInState().user.email)
+
+        store.commit('unsetUser')
+        expect(store.getters['user']).toBe(null)
```

Comment:
Same comment as above re: readability

Reply:
Since we're testing unsetUser I've removed the detailed checks on user.